modules/UI.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable max-classes-per-file */
+import Store from './store.js';
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const details = document.createElement('div');
+    const list = document.querySelector('#books');
+
+    details.style.backgroundColor = 'rgba(125, 125, 125, 0.3)';
+    details.innerHTML = `
+          <h5>"${book.title}" by <span>${book.author}</span></h5>
+          <input type="submit" class="remove" value="Remove">        
+        `;
+    list.appendChild(details);
+    const numBooks = list.children.length;
+    if (numBooks % 2 === 0) {
+      details.style.backgroundColor = 'rgb(255, 255, 255)';
+    }
+  }
```

Comment:
- [ ] The add book feature is currently not working as expected. The books are neither added to the localStorage nor are they displayed on the page. Kindly make it work 👍